src/main/java/com/minecolonies/coremod/client/model/ModelEntityDeliverymanMale.java

```diff
@@ -59,7 +66,14 @@ public void render(final Entity entity, final float limbSwing, final float limbS
     }
 
     @Override
-    public void setRotationAngles(final float limbSwing, final float limbSwingAmount, final float ageInTicks, final float netHeadYaw, final float headPitch, final float scaleFactor, final Entity entityIn)
+    public void setRotationAngles(
+                                   final float limbSwing,
+                                   final float limbSwingAmount,
+                                   final float ageInTicks,
+                                   final float netHeadYaw,
+                                   final float headPitch,
+                                   final float scaleFactor,
+                                   final Entity entityIn)
     {
         bipedHead.rotateAngleY = netHeadYaw / 57.29578F;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a "double" or "BigDecimal" instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2164)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 57.29578F to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)